Classes/TiMaterialCardViewProxy.m

```diff
@@ -0,0 +1,21 @@
+//
+//  TiMaterialCardViewProxy.m
+//  titanium-material
+//
+//  Created by brian garcía on 20/4/18.
+//
+
+#import "TiMaterialCardViewProxy.h"
+#import "TiMaterialCardView.h"
+#import "TiUtils.h"
+
+@implementation TiMaterialCardViewProxy
+
+
+- (void)setContentView:(id)viewProxy
+{
+    ENSURE_TYPE(viewProxy, TiViewProxy);
+    [[(TiMaterialCardView *)[self view] cardView] setContentView:[viewProxy view]];
```

Comment:
Does the layouting work properly? I would assume that some auto-sizing does not work out of the box.